### ISSUE TITLE
[Arcs] First Regent always takes trophies when Empire defends?

### ISSUE THREAD

**Issue by turmoilbyrd** [LederCards/cards#106]
This rule https://arcs.seiyria.com/#14.4.1.1-trophies (campaign rulebook p.11) says

> If the Empire is defending, ... the First Regent takes any destroyed pieces **of the Outlaw** as Trophies

I think this was written when it wasn't clear that a Regent can also battle the Empire.
Should this rule be extended to say

> ...the First Regent takes any destroyed pieces of the attacker as Trophies

?

related
- https://github.com/LederCards/cards/issues/57

**Comment by turmoilbyrd** [comment]
Personally I think it's a good idea to extend it. It is how I would have intuit it.

Plus, in the rare event when a First Regent attacks the Empire, they will likely not receive any trophies. (all of the destroyed pieces are Loyal and will just return to supply)

**Comment by yggdrasilsYeoman** [comment]
> > Hey Josh, I just realized a related minor hole. There currently is not a rule providing for taking Imperial ships as Trophies when destroying them through non-battle means, such as Warrior Golem. The base rules only cover taking Rival pieces as trophies through general destruction,...
> 
> But you take Blight destroyed by securing Blight Loom right? So it might be the base rule needs to be written more tightly but we are already playing the RAI


No, Blight Trophies are explicitly covered for all destruction purposes under the Blight section, p. 19: “Take Blight you destroy as Trophies.” That part is not limited to battle, so that’s what makes Blight Looms work.

**Comment by turmoilbyrd** [comment]
It makes no sense to define trophy-taking rules for every single NPC factions. It should just be catch-all i.e. "when you destroy non-Loyal pieces, take them as trophies".
